operations/model_admin.go

```diff
@@ -0,0 +1,34 @@
+package operations
+
+import (
+	"strings"
+
+	"github.com/mongodb/grip"
+	"github.com/pkg/errors"
+)
+
+// ClientAdminConf tracks information about the evergreen deployment
+// internally to support doing operations against specific app servers
+// individually, including administering background jobs, and
+// potentially self-orchestrating deploys.
+type ClientAdminConf struct {
```

Comment:
Could we name this something like ConnectionInfo? I don't think we'd ever want an entire admin config here, just how to connect to the server where the config is stored?

Reply:
I imagine that we'll want more admin-ish type things to do deployment orchestration (perhaps) in the future. Things like aws credentials, etc to handle pushing new builds to beanstalk or similar, maybe also names of amboy queues/etc.